src/seedu/addressbook/common/Utils.java

```diff
@@ -34,4 +34,15 @@ public static boolean elementsAreUnique(Collection<?> items) {
         }
         return true;
     }
+
+    /**
+    * Returns a new set of lower case string.
+    */
+    public static Set<String> convertSetStringToLowerCase(Set<String> items) {
```

Comment:
The header comment does not follow the convention. All non trivial methods should have javadoc format header comments. (First line is the description of method, followed by @param for specifying the parameters and @return for specifying the return values, if applicable)